src/Builder/ClassMetadataBuilder/ClassMetadataBuilder.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
```

Comment:
Here is the same. Please use 

`declare(strict_types=1);`

Reply:
++ 

Added config to PHPStorm